benthomasson/eda/tests/unit/requirements.txt

```diff
@@ -0,0 +1,3 @@
+aiohttp
```

Comment:
same as I commented here https://github.com/benthomasson/benthomasson-eda/pull/10#discussion_r946708278

Reply:
I'm trying to do it for kafka: https://github.com/benthomasson/benthomasson-eda/pull/12/files

Reply:
Based on the doc: https://docs.ansible.com/ansible/latest/dev_guide/testing_units.html, the requirements.txt has to be located at tests/unit directory